Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1251,7 +1253,9 @@ - (void)initializeSession {
     else if ([self.branchKey rangeOfString:@"key_test_"].location != NSNotFound) {
         [self.preferenceHelper logWarning:@"You are using your test app's Branch Key. Remember to change it to live Branch Key for deployment."];
     }
-    
+
+    [BNCSearchAdAttribution checkAttributionWithCompletion:nil];
```

Comment:
Hmmm - this isn't correct. I thought we talked about mimicking the functionality of Facebook's SDK call?